integration-tests/bot/client_cli.py

```diff
@@ -0,0 +1,328 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import subprocess
+import re
+import json
+
+PASSPHRASE = "123456"
+def run(cmd, args=None):
+    p = subprocess.Popen(cmd,
+                         stdin=subprocess.PIPE,
+                         stdout=subprocess.PIPE,
+                         stderr=subprocess.PIPE,
+                         universal_newlines=True,
+                         bufsize=0)
+    if isinstance(args, list) and args:
+        p.stdin.writelines("\n".join(args) + "\n")
+    elif args:
+        p.stdin.writelines("{}\n".format(args))
+    output, err = p.communicate()
+    if err and "using mock" not in err.lower():
+        raise Exception("stdout: {}, stderr: {}", output, err)
+    return remove_color(output)
+
+
+"""
+input 100.0100000 return 100 * 100000000 + 100000 = 1000100000
+"""
+def to_coin(balance):
```

Comment:
```suggestion
def to_coin(balance):
    """
    input 100.0100000 return 100 * 100000000 + 100000 = 1000100000
    """
```
function's docstring